inc/Ui/Search.php

```diff
@@ -584,29 +584,25 @@ protected function getFulltextResultsHTML($data, $highlight)
             if ($cnt !== 0) {
                 $resultHeader[] = $cnt . ' ' . $lang['hits'];
                 if ($num < FT_SNIPPET_NUMBER) { // create snippets for the first number of matches only
-                    $snippet = '<dd>' . ft_snippet($id, $highlight) . '</dd>';
-                    $lastMod = '<span class="search_results__lastmod">' . $lang['lastmod'] . ' ';
+                    $snippet = '<dd class="snippet">' . ft_snippet($id, $highlight) . '</dd>';
+                    $lastMod = $lang['lastmod'] . ' ';
```

Comment:
would it make sense to wrap this label into a span, so template authors can hide it or replace it by an icon?